recipes/kfserving.rb

```diff
@@ -0,0 +1,208 @@
+# Install KFServing and required tools
+
+# Istio
+
+directory "/home/#{node['kube-hops']['user']}/istio" do
+  owner node['kube-hops']['user']
+  group node['kube-hops']['group']
+  mode '0700'
+  action :create
+end
+
+bash 'install-istioctl' do
+  user node['kube-hops']['user']
+  group node['kube-hops']['group']
+  environment ({ 'HOME' => ::Dir.home(node['kube-hops']['user']) })
+  cwd ::Dir.home(node['kube-hops']['user'])
+  code <<-EOH
+    cd istio
+    istio_tar="istio-#{node['kube-hops']['istio_version']}-linux-amd64.tar.gz"
+    wget -nc #{node['kube-hops']['istio_url']}
+    tar zxf $istio_tar
+    rm -f $istio_tar
+    rm -f istio
+    ln -s istio-#{node['kube-hops']['istio_version']} istio
+    EOH
+  not_if { File.exist? "/home/#{node['kube-hops']['user']}/istio/istio-#{node['kube-hops']['istio_version']}/bin/istioctl" }
```

Comment:
It was a mistake to put some of the Yaml file on the `home` dir, let's not continue with it. All Hopsworks services are installed in `/srv/hops` - please move istio there as well.